engine/graphics/src/webgpu/graphics_webgpu.cpp

```diff
@@ -1176,7 +1176,13 @@ static HContext WebGPUNewContext(const ContextParams& params)
 static bool WebGPUIsSupported()
 {
     TRACE_CALL;
-    return true;
+    return MAIN_THREAD_EM_ASM_INT({
```

Comment:
Use `MAIN_THREAD_EM_ASM_INT` here just in case if web build will support multithreading in the future. Because `Module.hasWebGPUSupport()` manipulates DOM tree (creates temp canvas and tries obtain context) so need to run in main thread.